crates/ra_mbe/src/lib.rs

```diff
@@ -50,6 +52,38 @@ pub(crate) struct Rule {
     pub(crate) rhs: tt::Subtree,
 }
 
+// Find the max token id inside a subtree
```

Comment:
Please use a doc comment.

Reply:
Facepalm. Yeah, let me fix this in future PR, thanks!